core/modules/xrdsvc/SsiProvider.h

```diff
@@ -41,28 +53,29 @@ class SsiProviderServer : public XrdSsiProvider
 {
 public:
 
-    virtual XrdSsiService *GetService(XrdSsiErrInfo& eInfo, char const* contact, int oHold=256) {
+    virtual XrdSsiService *GetService(XrdSsiErrInfo& eInfo,
+                                      char const* contact, int oHold=256) {
         return _service.get();
     }
 
     virtual bool Init(XrdSsiLogger* logP, XrdSsiCluster* clsP, char const* cfgFn,
-            char const* parms, int argc, char **argv) {
-        _service = std::unique_ptr<SsiService>(new SsiService(logP));
-        return true;
-    }
+                      char const* parms, int argc, char** argv);
 
-    virtual rStat QueryResource(char const*rName, char const* contact=0) {
-        // Not called on this object but part of pure virtual interface,
-        // so we need to provide at least this dummy implementation.
-        return isPresent;
-    }
+    virtual rStat QueryResource(char const* rName, char const* contact=0);
+
+                   SsiProviderServer() : _cmsSsi(0), _logSsi(0) {}
```

Comment:
I totally agree that we should not care about white space in C++ but this line is totally mis-aligned :)